extensions/rocksdb-repos/FlowFileRepository.cpp

```diff
@@ -152,7 +153,10 @@ void FlowFileRepository::prune_stored_flowfiles() {
         // we find the connection for the persistent flowfile, create the flowfile and enqueue that
         std::shared_ptr<core::FlowFile> flow_file_ref = std::static_pointer_cast<core::FlowFile>(eventRead);
         eventRead->setStoredToRepository(true);
-        search->second->put(eventRead);
+        auto connection = std::static_pointer_cast<Connection>(search->second);
```

Comment:
if the #807 is merged Processors might also be FlowFile containers (like `BinFiles`) and those files should also be restored